WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1510,32 +1544,40 @@ private extension ReaderStreamViewController {
         let buttonTitle = buttonTitleForTopic(topic)
         let imageName = ReaderHelpers.topicIsFollowing(topic) ? readerEmptyImageName : nil
 
-        configureAndDisplayResultsStatus(title: response.title, subtitle: response.message, buttonTitle: buttonTitle, imageName: imageName)
+        configureResultsStatus(title: response.title, subtitle: response.message, buttonTitle: buttonTitle, imageName: imageName)
+        displayResultsStatus()
     }
 
     func displayNoConnectionView() {
-        configureAndDisplayResultsStatus(title: ResultsStatusText.noConnectionTitle, subtitle: noConnectionMessage())
+        configureResultsStatus(title: ResultsStatusText.noConnectionTitle, subtitle: noConnectionMessage())
+        displayResultsStatus()
     }
 
-    func configureAndDisplayResultsStatus(title: String,
-                                          subtitle: String? = nil,
-                                          buttonTitle: String? = nil,
-                                          imageName: String? = nil,
-                                          accessoryView: UIView? = nil) {
+    func configureResultsStatus(title: String,
+                                subtitle: String? = nil,
+                                buttonTitle: String? = nil,
+                                imageName: String? = nil,
+                                accessoryView: UIView? = nil) {
 
+        (accessoryView as? WPAnimatedBox)?.animate()
```

Comment:
@ScoutHarris - For some reason, after the changes made to the controller, the animated box didn't animate anymore.
I solved it by calling `animate()` directly here, but I noticed that this is not necessary anywhere else in the app.

I'm not sure about the reason of that, nor if this is a good solution.
Any input is welcome! 😁

Reply:
@etoledom please note that if you explicitly call `animate()`, you'll also want to call `suspendAnimation` to prevent recurrences of #10047.

Reply:
Interesting. Thanks for the heads up! 
I'll give it a second revision. I'm not confortable calling it directly.
The PR you mentioned might give me some clues on how to handle it correctly.

Thanks again! :)